superset/jinja_context.py

```diff
@@ -338,6 +347,8 @@ def get_template_processor(
         template_processor = template_processors.get(
             database.backend, BaseTemplateProcessor
         )
+    elif feature_flag_manager.is_feature_enabled("CHEVRON_TEMPLATE_PROCESSING"):
```

Comment:
Might be better to nest this conditional under if `ENABLE_TEMPLATE_PROCESSING`, sort of as a sub option after the user turns tempate processing `on`

Reply:
Maybe we should rename `ENABLE_TEMPLATE_PROCESSING` to `JINJA_TEMPLATE_PROCESSING` instead (albeit a breaking change)?

Reply:
Right, clearly there's some cleanup to be done here, but also need to maintain backward compatibility. I think that's the easy portion of this PR, the real question is around the viability of `chevron` as a secure templating backend.

Reply:
I would propose calling the feature flag `TEMPLATE_PROCESSING`, and then adding a config parameter `TEMPLATE_PROCESSOR: Optional[TemplateProcessorType] = TemplateProcessorType.CHEVRON`. Similar to how other feature flagged options do (see e.g. `THUMBNAIL_SELENIUM_USER`, `THUMBNAIL_CACHE_CONFIG` etc).